android/build.gradle

```diff
@@ -280,5 +343,8 @@ dependencies {
         // Common functionality used across multiple media libraries
         implementation "androidx.media3:media3-common:$media3_version"
     }
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-}
+
+    if (isReactNativeVersionBetween(reactNativeVersion, minReactNativeVersion, maxReactNativeVersionExclusive)) {
```

Comment:
Here you are blocking build for version < minVersion . I don't think this is really expected...